### DISCUSSION TITLE
failure in pet-freesurfer pipeline

### DISCUSSION THREAD

**Discussion by takaakiyoshim** [aramis-lab/clinica#870] [Q&A]
Hi,
I tried to run pet-freesurfer pipeline for one subject, but it failed with the error messeage as bellow.
 
```
ERROR:Clinica does not handle 4D volumes for sub-ADNI002S0413 | ses-M132

```

Indeed, input PET data in BIDS directory has formally 4 dimensions, but 4th dimension is one as bellow.

```
 fslinfo sub-ADNI002S0413_ses-M132_trc-18FAV45_pet.nii.gz
data_type	FLOAT32
dim1		128
dim2		128
dim3		90
dim4		1
datatype	16
pixdim1		2.000000
pixdim2		2.000000
pixdim3		2.000000
pixdim4		0.000000
cal_max		0.000000
cal_min		0.000000
file_type	NIFTI-1+
```

Let me know how to solve this issue.

Best regards,
Yoshimoto

**Comment by alexandreroutier** [comment]
Hi!

Thanks for your interest in Clinica :)

You are right, your particular case was not taken into account when Clinica had to discard longtudinal acquistions of PET data. I have opened an issue https://github.com/aramis-lab/clinica/issues/871 and you can manually modify the section of the pipeline you want to run.

Best,
Alexandre

**Reply by takaakiyoshim** [reply]
Thank you for your quick response !
I would apply your suggestion to my case.
